gamemode/modules/base/sv_gamemode_functions.lua

```diff
@@ -308,6 +308,7 @@ local DrpCanHear = {}
 -- proxy function to take load from PlayerCanHearPlayersVoice, which is called a quadratic amount of times per tick,
 -- causing a lagfest when there are many players
 local function calcPlyCanHearPlayerVoice(listener)
+    if not IsValid(listener) then return end
     DrpCanHear[listener] = DrpCanHear[listener] or {}
     if listener:IsBot() then return end
```

Comment:
Why not merge the to if statements?

Reply:
Because bots need to be added to the list, as I stated above.
They are considered by PlayerCanHearPlayersVoice.

Reply:
Oh